docs/api.md

```diff
@@ -221,6 +221,16 @@ render_inline(MyComponent.new)
 assert_text("Hello, World!")
 ```
 
+If no `component` is provided, `render_inline` executes the block in the view context:
+
+```ruby
+render_inline do
+  render(MyComponent.new)
+end
+
+assert_text("Hello, World!")
+```
+
```

Comment:
I think these changes should be removed ❤️ 

The API docs are auto-generated anyways.

Reply:
Yeah... I thought the docs were generated and committed back to the repo, but nothing happened this time? Maybe I'm thinking of a different repo.